client/me/purchases/cancel-purchase/index.jsx

```diff
@@ -349,13 +404,6 @@ class CancelPurchase extends Component {
 							cancellationFeatures={ cancellationFeatures }
```

Comment:
Note for future iterations @fditrapani - I think it would be great to get Jetpack moved to use this cancellation features list display instead of showing the features that will be lost in a modal.

One thing that I like about what Jetpack does though is showing the features lost before removal - which is helpful to show what's about to stop working.

![Screenshot 2025-06-26 at 5 12 05 PM](https://github.com/user-attachments/assets/004731f1-eb5a-479b-82d9-f88ffe1b006b)

Reply:
Absolutely! All good points. The new design will present the features on purchase settings screen, before the cancellation and to hopefully prevent people from pressing the cancel in the first place:
![image](https://github.com/user-attachments/assets/8139c514-fe2a-4b66-9b0b-674da9728c41)

The we'll echo the features they'll lose on the cancel screen.
![image](https://github.com/user-attachments/assets/06c11a41-6781-45f5-ad4f-dddca72dce34)